datasets/il/wmd_banned/crawler.py

```diff
@@ -0,0 +1,244 @@
+from zavod import Context
+from typing import List, Optional, Tuple
+from rigour.mime.types import XLSX
+import openpyxl
+from zavod import helpers as h
+import shutil
+import re
+import datetime
+
+
+# Hebrew to Gregorian month mapping
+HEBRW_MONTHS = {
+    "ינואר": "January",
+    "פברואר": "February",
+    "מרץ": "March",
+    "אפריל": "April",
+    "מאי": "May",
+    "יוני": "June",
+    "יולי": "July",
+    "אוגוסט": "August",
+    "ספטמבר": "September",
+    "אוקטובר": "October",
+    "נובמבר": "November",
+    "דצמבר": "December",
+}
+
+
+def hebrew_to_datetime(date_str: str):
+    """
+    Convert a Hebrew date string to a Normal datetime.date object.
+    """
+    if not date_str:
+        return None
+
+    if isinstance(date_str, int):
+        return datetime.date(date_str, 1, 1)
+
+    date_str = re.sub("\n", "", date_str)
+    for heb_month, eng_month in HEBRW_MONTHS.items():
+        date_str = re.sub(heb_month, f"{eng_month} ", date_str, flags=re.IGNORECASE)
+        date_str = re.sub("ב", "", date_str, flags=re.IGNORECASE)
+
+    date_fmt = ["%d %B %Y", "%d %B. %Y"]
+    for fmt in date_fmt:
+        try:
+            date_obj = datetime.datetime.strptime(date_str, fmt)
+            return date_obj.date()
+        except ValueError:
+            pass
```

Comment:
can you replace most or all of this with this new approach? https://zavod.opensanctions.org/best_practices/dates_meta/

Reply:
Sure, it seems the lookup uses lower case so we have to add the property or metadata as lowercase.
I'm using lowercase for the property in my case